modules/tests/shared/src/test/scala/gsp/math/skycalc/package.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gsp.math
+
+import java.time.ZoneId
+
+package object skycalc {
+  val GN =
```

Comment:
It seems this would be generally useful, are they going to be in `gsp-core`?

Reply:
`gsp-core` already has `Site`, and instances for `GN` and `GS`. But we don't have access to those from here. These here are only used for testing against known results

Reply:
👍